ccdb/fields.html

```diff
@@ -154,7 +154,13 @@
 			<td data-label="Field name">Consumer disputed?</td>
 			<td data-label="Description">Whether the consumer disputed the company’s response</td>
 			<td data-label="Data type">plain text</td>
-			<td data-label="Notes">yes/no</td>
+			<td data-label="Notes">
+				<strong>yes</strong>
+				<br/>
+				<strong>no</strong>
+				<br/>
+				<strong>N/A:</strong> The Bureau discontinued the consumer dispute option on April 24, 2017.
+			</td>
```

Comment:
Though it looks a little weird with no explanation after the "yes" and "no" values, let's follow the pattern from the "Consumer consent provided" row and capitalize + space these notes like so:

```html
<td data-label="Notes">
  <strong>Yes</strong>
  <br>
  <br>
  <strong>No</strong>
  <br>
  <br>
  <strong>N/A:</strong> The Bureau discontinued the consumer dispute option on April 24, 2017.
</td>
```